numpy/random/mtrand/mtrand.pyx

```diff
@@ -4653,6 +4673,191 @@ cdef class RandomState:
+            than ``sum(colors)``.
+        size : int or tuple of ints, optional
+            Output shape.  If the given shape is, e.g., ``(m, n, k)``,
+            then ``m * n * k`` samples are drawn, and the return value has
+            shape ``(m, n, k, len(colors))``.  If `size` is an integer,
+            the output has shape ``(size, len(colors))``.  Default is None,
+            in which case a single sample is returned as an array with shape
+            ``(len(colors),)``.
+        method : string, optional
+            Specify the algorithm that is used to generate the sample.
+            Must be 'count' (the default) or 'iterative'.  See the Notes
+            for a description of the methods.
+
+        Returns
+        -------
+        sample : ndarray
+            Sample drawn from the multivariate hypergeometric distribution.
+
+        See Also
+        --------
+        hypergeometric : Draw samples from the (univariate) hypergeometric
+            distribution.
+
+        Notes
+        -----
+        The two methods do not return the same sequence of samples.
+
+        The "count" algorithm is roughly equivalent to the following numpy
+        code::
+
+            choices = np.repeat(np.arange(len(colors)), colors)
+            selection = np.random.choice(choices, nsample, replace=False)
+            sample = np.bincount(selection, minlength=len(colors))
+
+        The "count" algorithm uses a temporary array with length
+        ``sum(colors)``.
+
+        The "iterative" algorithm generates a single sample by using repeated
+        calls to the univariate hypergeometric sampler.  It is roughly
+        equivalent to::
+
+            sample = np.zeros(len(colors), dtype=np.int)
+            # `remaining` is the cumulative sum of `colors` from the last
+            # element to the first; e.g. if `colors` is [3, 1, 5], then
+            # `remaining` is [9, 6, 5].
+            remaining = np.cumsum(colors[::-1])[::-1]
+            for i in range(len(colors)-1):
+                if nsample < 1:
+                    break
+                sample[i] = np.random.hypergeometric(colors[i], remaining[i+1],
+                                                     nsample)
+                nsample -= sample[i]
+            sample[-1] = nsample
+
+        For some cases, the 'iterative' algorithm may be significantly faster
+        than the 'count' algorithm.
+
+        .. versionadded:: 1.14.0
+
+        Examples
+        --------
+        >>> colors = [16, 8, 4]
+        >>> np.random.seed(123)
+        >>> np.random.multivariate_hypergeometric(colors, 6)
+        array([4, 1, 1])
+        >>> np.random.multivariate_hypergeometric(colors, 6, size=3)
+        array([[3, 1, 2],
+               [3, 2, 1],
+               [2, 3, 1]])
+        >>> np.random.multivariate_hypergeometric(colors, 6, size=(2, 2))
+        array([[[4, 1, 1],
+                [4, 2, 0]],
+
```

Comment:
The blank lines mess up the document rendering in the examples, best to remove them.